src/Microsoft.ML.Data/DataLoadSave/TransformWrapper.cs

```diff
@@ -168,26 +168,4 @@ public SchemaShape GetOutputSchema(SchemaShape inputSchema)
             return SchemaShape.Create(transformer.GetOutputSchema(fakeSchema));
         }
     }
-
-    /// <summary>
-    /// Estimator for untrained wrapped transformers.
-    /// </summary>
-    public abstract class TrivialWrapperEstimator : TrivialEstimator<TransformWrapper>
```

Comment:
>TrivialWrapperEstimator [](start = 26, length = 23)

where did this go ?

Reply:
ah i see. no usages in code :)

---
In reply to: [259595477](https://github.com/dotnet/machinelearning/pull/2701#discussion_r259595477) [](ancestors = 259595477)

Reply:
To valhalla!